BangazonWebApp/BangazonWebApp/Models/Employees.cs

```diff
@@ -0,0 +1,41 @@
+﻿//Author: Lauren Richert
+//Purpose: To reference the Employees table and the values
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonWebApp.Models
+{
+    public class Employees
+    {
+        public int Id { get; set; }
+
+        [Required]
+        [Display(Name = "First Name")]
+        public string FirstName { get; set; }
+
+        [Required]
+        [Display(Name = "Last Name")]
+        public string LastName { get; set; }
+
+        [Required]
+        [Display(Name = "Department")]
+        public int DepartmentsId { get; set; }
+
+        [Required]
+        [Display(Name = "Assigned Computer")]
+        public int ComputersId { get; set; }
```

Comment:
According to Ticket #5, you need to see their current computer when you view the employee's details- not the computer's id. This property should be called `CurrentComputer` or something similar and should be typed as a `<Computer>`